homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -17,6 +17,7 @@
     AsyncPlugableSwitchMeasuring,
     AsyncPresenceDetectorIndoor,
     AsyncRoomControlDeviceAnalog,
+    AsyncTemperaturDifferenceSensor2,
```

Comment:
Seems to have a typo upstream? Temperature is spelled incorrect

```suggestion
    AsyncTemperatureDifferenceSensor2,
```

Reply:
Well yes, you find everything. Sorry for that!
Same spelling as in the Libray. Can i close this PR and open a new one ,which includes two changes? 
1. Fixing this typo
2. Add another device?

Reply:
I would just make a quick fix in the library, bump the version of the library and adjust this PR?

Can also be done later, as in, this is not exposed anyways. Just noticed it.